### DISCUSSION TITLE
Issue when creating Policy Definition for - Deploy Virtual Machine Auto Shutdown Schedule

### DISCUSSION THREAD

**Discussion by paul-dooney-daa** [Azure/Enterprise-Scale#1467] [General] [answered]
Hi All, I have an issue when creating a Policy Definition for - 'Deploy Virtual Machine Auto Shutdown Schedule'. I get the following error:

"The policy '********************************' has defined parameters 'time,timeZoneId,EnableNotification,NotificationEmailRecipient,NotificationWebhookUrl' which are not used in the policy rule. Please either remove these parameters from the definition or ensure that they are used in the policy rule."

Has anyone experienced this before? Any help greatly appreciated!

**Comment by jtracey93** [comment, marked as answer]
How are you trying to deploy that @paul-dooney-daa 

As this sounds like an escaping issue.

Try copying the definition from the JSON section here: https://www.azadvertizer.net/azpolicyadvertizer/Deploy-Vm-autoShutdown.html and that should have the correct escaping 👍

**Reply by paul-dooney-daa** [reply]
Thank you for the response @jtracey93. I was copying the definition from https://github.com/Azure/Enterprise-Scale/blob/main/src/resources/Microsoft.Authorization/policyDefinitions/Deploy-Vm-autoShutdown.json. Once I copied it form azadvertizer it worked. Cheers!
